wolfhsm/wh_common.h

```diff
@@ -80,7 +80,7 @@ typedef struct {
 
 
 /** Manifest storage  */
-
+#if 0
```

Comment:
did you mean to leave this in?

Reply:
This was NE's first struct that we talked about.  I'll remove it.